server/src/main.rs

```diff
@@ -147,6 +148,85 @@ pub async fn query_vertiports(
     }
 }
 
+/// Requests the cost of an itinerary from svc-pricing
+///
+/// # Arguments
+/// * _flight - The flight plan to obtain the cost of
+/// * client  - The active client for the svc-pricing server
+///
+/// # Returns
+/// A tuple of price and currency type
+async fn customer_cost(
```

Comment:
Maybe rename to `get_base_pricing`?

It's a good practice to start with a verb for a function.

Reply:
It's actually idiomatic rust to not use "get" for getters, but to use "set" for setters:

https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis

Will change to "base_pricing"